src/pydvl/value/semivalues.py

```diff
@@ -198,6 +199,15 @@ def compute_generic_semivalues(
         coefficient: The semi-value coefficient
         done: Stopping criterion.
         batch_size: Number of marginal evaluations per single parallel job.
+        skip_converged: Whether to skip marginal evaluations for indices that
```

Comment:
How about printing a warning as well?

Reply:
🤔 About what? There is no change in behaviour from previous versions. Or do you mean we should transition to a default of `true` and warn that this will be so?

Reply:
I mean a warning if ones uses a different Stopping Criterion than advised.

Reply:
No, I don't think so. This parameter doesn't change what's recommended or isn't, it only highlights an issue with global stopping criteria. There should be good documentation about what the different criteria do, but no warnings annoying people who possibly know what they are doing.